packages/server/test/signup.integration.ts

```diff
@@ -24,10 +29,28 @@ describe('SignUp Integration', () => {
       expect(res.status).toBe(400);
     });
 
-    it('returns 409 when user exists', async () => {
+    it('just register user if user exists', async () => {
```

Comment:
What do you mean by this?

Reply:
Register user in course if user exists without creating new user.